apps/api/src/workspace/service/workspace.service.ts

```diff
@@ -385,6 +385,103 @@ export class WorkspaceService {
     return { projects, environments, secrets, variables }
   }
 
+  /**
+   * Gets all workspaces of a user, paginated.
+   * @param user The user to get the workspaces for
+   * @param page The page number to get
+   * @param limit The number of items per page to get
+   * @param sort The field to sort by
+   * @param order The order to sort in
+   * @param search The search string to filter by
+   * @param isAccepted The invitation status to filter by
+   * @returns The workspace invitations of the user, paginated, with metadata
+   */
+  async getAllWorkspaceInvitations(
+    user: User,
+    page: number,
+    limit: number,
+    sort: string,
+    order: string,
+    search: string,
+    isAccepted: boolean | undefined
```

Comment:
Don't bother about this field. We always want to fetch the ones that have `isInvitationAccepted` set to `false`